src/pages/Quiz/index.js

```diff
@@ -29,8 +45,15 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const Quiz = () => {
+const Quiz = (props) => {
   const classes = useStyles();
+
+  const { questions = [] } = props;
+
+  useEffect(() => {
+    props.fetchQuestions();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why do you need to disable eslint here?

Reply:
fixed. update will be in next commit

Reply:
How have you fixed it?
If you can't fix it, tell me a reason.